packages/terra-form/src/Control.jsx

```diff
@@ -97,6 +97,7 @@ const Control = ({
 
   const controlInputClasses = cx([
     'input',
+    (customProps.required ? 'required' : ''),
```

Comment:
ClassNames allows conditional checks first class for classes. 

Example: https://github.com/cerner/terra-core/blob/master/packages/terra-button/src/Button.jsx#L88